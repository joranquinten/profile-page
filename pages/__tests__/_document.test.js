import { shallow } from 'enzyme';

import MyDocument from '../_document';

describe('<MyDocument />', () => {
  it('should render without crashing', () => {
    expect(shallow(<MyDocument />)).toMatchSnapshot();
  });
});
