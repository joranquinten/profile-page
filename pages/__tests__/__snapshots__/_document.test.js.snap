// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyDocument /> should render without crashing 1`] = `
<html
  lang="nl"
>
  <Head />
  <body>
    <Main />
    <NextScript />
  </body>
</html>
`;
